### DISCUSSION TITLE
Exterior/Math/P%26S/7/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#10] [General]
# Exterior/Math/P%26S/7/

https://savia7582.github.io/Exterior/Math/P%26S/7/

**Comment by LH-0124** [comment]
例4答案可能有错，两个独立随机变量方差应该相加

**Comment by NoughtQ** [comment]
“二 三个抽样分布“里的 “F分布”的标号应为3，且公式上面错写成“t分布”了
